nakama/lib/src/nakama_client/nakama_client.dart

```diff
@@ -48,13 +48,13 @@ abstract class NakamaBaseClient {
   /// - [email] The email address of the user.
   /// - [username] A username used to create the user.
   /// - [password] The password for the user.
-  /// - [create] If the user should be created when authenticated.
+  /// - [create] If the user should be created when authenticated (true by default).
   /// - [vars] Extra information that will be bundled in the session token.
   Future<model.Session> authenticateEmail({
     String? email,
     String? username,
     required String password,
-    bool create = false,
```

Comment:
Could we keep this as-is?

Reply:
Just changed it to match the dotnet client, if to be reversed i need to update the tests to create the account when calling authenticate functions.